test-harness/specs/parameters-ac6.only_oas3.txt

```diff
@@ -0,0 +1,34 @@
+====test====
+When I send a request to an operation
+And the operation has a required query param
+And in the request I sent that param is present
+And the param I sent was not valid style-wise (ie form instead of pipeDelimited)
+Then I should get a 422 response back
+And the payload of the response should be in application/problem+json content-type
+====spec====
+openapi: 3.0.2
+paths:
+  /pet:
+    get:
+      parameters:
+        - in: query
+          name: name
+          required: true
+          style: pipeDelimited
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        '200':
+          description: ok
+          content:
+            application/json:
+              example: "hello"
+====server====
+mock -p 4010
+====command====
+curl -sIXGET http://127.0.0.1:4010/pet?name=abc,xyz
```

Comment:
@philsturgeon @lag-of-death Why is this supposed to fail? The `pipeDelimited` serializer can still correctly serialize this query string.

`"abc,xyz".split('|') = abc,xyz`

Since the schema is just requesting an array of string, the final result is `['abc,xyz']` which is still a valid array of strings.

Reply:
Ooo good point. Could we add "minItems: 2" to the schema so we get an error if there is only one item?